data/src/main/java/com/example/data/DataApplication.java

```diff
@@ -0,0 +1,53 @@
+package com.example.data;
+
+/**
+ * Create app that should support create, read, update and delete operations for some entity
+ * Use Spring Data module
+ * Don't use Spring Data REST starter
+ */
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class DataApplication {
+
+	private static final Logger log = LoggerFactory.getLogger(DataApplication.class);
+
+	public static void main(String[] args) {
+		SpringApplication.run(DataApplication.class);
+	}
+
+	@Bean
```

Comment:
Usually that is bad idea to put some extra code into the Application. Keep it simple. Instead feel free to define separate `@Component` with appropriated interface to run console apps

Reply:
it is just a demo, the service component will be created